rskj-core/src/main/java/org/ethereum/rpc/Web3.java

```diff
@@ -64,4 +65,5 @@
     PeerScoringInformation[] sco_peerList();
     String[] sco_bannedAddresses();
     PeerScoringReputationSummary sco_reputationSummary();
+    PeerScoringInformation[] sco_clearPeerScoring(String id);
```

Comment:
shouldn't this method return `void`, same as eg. `sco_banAddress`?

Reply:
I thought returning the list could be helpful, but it can be retrieved with `sco_peerList` so let's keep consistency, will return `void` 👍